### ISSUE TITLE
Selecting range selector button should inform screen reader user of change.

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#14438]
#### Expected behaviour
When selecting a range selector button, screen reader users should be informed that something has changed.

#### Actual behaviour
There is no notification to screen reader users after pressing the buttons.

#### Live demo with steps to reproduce
http://jsfiddle.net/gh/get/jquery/1.7.2/highcharts/highcharts/tree/master/samples/stock/accessibility/accessible-stock/

#### Product version
Highstock v8.2.2
